### PR TITLE
Create GRUNT.md

### LINKED ISSUE #1140: GRUNT.md
Developers may benefit from having a list of grunt commands traveling with the core framework. Yes, `grunt help` lists all commands, but some developers unfamiliar with grunt may not know this.

I'm not firm in my opinion that this additional documentation is needed. But am offering it in case others think it is helpful.

### PR DESCRIPTION
**Pull request by chucklorenz**
fixes #1140

### CODE REVIEW

**Comment by oliverfoster** [comment]
+1

**Comment by moloko** [comment]
+1

**Comment by hrajotia** [comment]
great doc @chucklorenz, just one more thing, can we document few other command line options too?

e.g. (menu, theme, port)
```
grunt build --menu=adapt-contrib-boxMenu --theme=adapt-contrib-vanilla
grunt server --port=8080
```

Thanks

**Comment by chucklorenz** [comment]
Yes @hrajotia , happy to include those options. Happy to include others I may have overlooked --just let me know.

**Comment by moloko** [comment]
+1

**Comment by lc-thomasberger** [comment]
+1

there is also the option to add a host, so that you can access the course from your network. defaults to localhost.
`grunt server --host=0.0.0.0`

File: GRUNT.md [hunk 1]
```diff
@@ -7,11 +7,16 @@ All Adapt grunt commands are run from the root directory of the Adapt course whe
 Lists out all available tasks along with descriptions. Useful to quickly look up what's available without having to leave the command line. <br/>_**Note**: Simply running `grunt` will have the same effect._ 
 
 **build**  
-`grunt build [--languages=xx,yy]`  
-Compiles all files in `src` to create a production-ready minified/compressed version of your course. Includes a variety of error-checking tasks. Add the "languages" flag to limit which language/s should be built, e.g, `grunt build --languages="de,fr"`.  
+`grunt build [--languages=xx,yy] [--menu=xx] [--theme=xx] [--sourcedir=xx] [--outdir=xx]`
```
Comment (hrajotia) [inline GRUNT.md]: small typo here.. should be `--outputdir`

File: GRUNT.md [hunk 2]
```diff
@@ -7,11 +7,16 @@ All Adapt grunt commands are run from the root directory of the Adapt course whe
...
+Compiles all files in `src` to create a production-ready minified/compressed version of your course. Includes a variety of error-checking tasks.  
+`--languages` Specifies which language folder/s should be built. Can take any value that matches the name of an existing */src/course/* subfolder. Defaults to `en`.  
+`--menu` If more than one menu is present in the */src/menu* folder, specifies the name of the menu to include in the build.  
+`--theme` If more than one theme is present in the */src/theme* folder, specifies the name of the theme to include in the build.  
+`--sourcedir` Specifies the absolute path to parent folder of the code to be processed. Defaults to the path to */src*.  
+`--outputdir`  Specifies the absolute path to parent folder where the build will be written. Defaults to the path to */build*. 
 
 **dev**  
-`grunt dev`  
+`grunt dev [--languages=xx,yy] [--menu=xx] [--theme=xx] [--sourcedir=xx] [--outdir=xx]`
```
Comment (hrajotia) [inline GRUNT.md]: small typo here.. should be `--outputdir`

**Comment by taylortom** [comment]
Can we also make sure that any changes are also made to https://github.com/adaptlearning/adapt_framework/wiki/Compiling%2C-testing-and-deploying-your-Adapt-course (not suggesting this is a task for you @chucklorenz)

**Comment by chucklorenz** [comment]
I added the two new translate tasks (knowing the tasks have not yet been released) to [Compiling, testing and deploying your Adapt course](https://github.com/adaptlearning/adapt_framework/wiki/Compiling,-testing-and-deploying-your-Adapt-course) (and cleaned up some typos, grammars, and formatting).  Others that are missing? The page could benefit from the addition of command options.

**Comment by hrajotia** [comment]
+1 - awesome.

**Comment by oliverfoster** [comment]
+1

**Pull request merged**